addons/stock_account/tests/test_account_move.py

```diff
@@ -362,3 +362,55 @@ def test_cogs_account_branch_company(self):
         })
 
         self.assertEqual(bill.invoice_line_ids.account_id, test_account)
+
+
+@tagged("post_install", "-at_install")
+class TestValuatedStockMove(TestAccountMoveStockCommon):
```

Comment:
Couldn't/shouldn't this test be part of `test_lot_valuation.py`? Could reuse some of its class setup values too (product/lot), unless you really want a lovely product :sparkles: